db/migrations/0012.add_mivideo_tables.py

```diff
@@ -0,0 +1,18 @@
+'''
+Migrations for MiVideo tables
+'''
+
+from yoyo import step
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS mivideo_media_started_hourly (
+            id BIGINT AUTO_INCREMENT NOT NULL UNIQUE,
+            event_hour_utc VARCHAR(20) NOT NULL,
+            course_id BIGINT NOT NULL,
```

Comment:
Could this be just `INTEGER`? I've been using `INTEGER` for most Canvas IDs elsewhere unless values get larger than 6 digits

Reply:
Can't they get quite big, like those beginning with `1770000…`?

Reply:
Those are the Unizin IDs. I am only seeing the standard 6 or fewer Canvas IDs in your table.

Reply:
OK.  I thought the longer IDs were still desired, as mentioned in the earlier comment: https://github.com/tl-its-umich-edu/course-inventory/pull/76#discussion_r408346194

So, yeah, I can make this `INTEGER`.

Reply:
No worries, I thought I explained in a follow-up to that comment that we aren't using the long 17-digit IDs anymore, so the six-digit Canvas IDs you have should be fine.